conf/flight_plans/rotorcraft_guided_flightplan.xml

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
+
+<flight_plan alt="1.0" ground_alt="0" lat0="51.990634" lon0="4.376789" max_dist_from_home="20" name="Rotorcraft Optitrack (Delft)" security_height="0.3">
+  <header>
+
+#include "autopilot.h"
```

Comment:
This include shouldn't be needed anymore, I made it available by default as it is often used.
Could you also fix the indentation of the rest of the file ?